IdentityManagement/Sessions/session.ts

```diff
@@ -0,0 +1,44 @@
+const express = require('express')
+const app = express()
+const port = 3000
+const axios = require('axios');
+const jwt = require("jsonwebtoken")
+
+const jwtKey = "my_secret_key"
+const jwtExpirySeconds = 300
+
+
+app.get('/token', async (req, res) => {
+
+    const userId = req.body.userId
```

Comment:
I think you need to configure body parsing before accessing the request body. See http://expressjs.com/en/resources/middleware/body-parser.html